index.html

```diff
@@ -0,0 +1,611 @@
+            <figure>
+              <img src="/img/wall-street-journal-logo.png" alt="Logo of Wall Street Journal" title="Wall Street Journal">
+            </figure>
+          </li>
+          <li>
+            <figure>
+              <img src="/img/the-guardian-logo.png" alt="Logo of the Guardian" title="the Guardian">
+            </figure>
+          </li>
+          <li>
+            <figure>
+              <img src="/img/verge-logo.png" alt="Logo of The Verge" title="The Verge">
+            </figure>
+          </li>
+          <li>
+            <figure>
+              <img src="/img/tech-crunch-logo.png" alt="Logo of TechCrunch" title="TechCrunch">
+            </figure>
+          </li>
+          <li>
+            <figure>
+              <img src="/img/lifehacker-logo.png" alt="Logo of lifehacker" title="lifehacker">
+            </figure>
+          </li>
+          <li>
+            <figure>
+              <img src="/img/mashable-logo.png" alt="Logo of Mashable" title="Mashable">
+            </figure>
+          </li>
+        </ul>
+      </section>
+      <section data-section="app-support" id="support">
+        <h1>App Support</h1>
+        <div class="l-flex-container-app-support">
+          <ul>
+            <li class="css-flag">
+              <div class="css-flag__image">
+                <span class="icon-location"></span>
+              </div>
+              <div class="css-flag__body">
+                <h2>Address</h2>
+                <p><a href="#">121 King Street, Melbourne<br>Victoria, Australia 3000</a></p>
+              </div>
+            </li>
+            <li class="css-flag">
+              <div class="css-flag__image">
+                <span class="icon-phone"></span>
+              </div>
+              <div class="css-flag__body">
+                <h2>Free Helpline</h2>
+                <p><a href="tel:001800345333700">(001) 800 345  333700</a></p>
+                <p><a href="tel:002600345333700">(002) 600 345  333700</a></p>
+              </div>
+             </li>
+            <li class="css-flag">
+              <div class="css-flag__image">
+                <span class="icon-envelope"></span>
+              </div>
+              <div class="css-flag__body">
+                <h2>Email</h2>
+                <p><a href="mailto:info@appbox.com">info@appbox.com</p>
+                <p><a href="mailto:feedback@appbox.com">feedback@appbox.com</a></p>
+              </div>
+            </li>
+          </ul>
+          <form>
+             <p>Nullam id odio quis risus placerat aliquam. Cum sociis natoque penatibus et magnis dis part urient montes nascetur ridiculus mus curabitur tellus lacus. </p><br>
+            <input type="text" name="name" placeholder="Full Name">
+            <input type="email" name="email" placeholder="Email">
+            <br>
+            <textarea name="message" placeholder="Message"></textarea>
+            <br>
+            <button class="btn-small" type="submit">Send</button>
```

Comment:
The inputs here should probably have associated `<labels>`, even if you hide them from view, for accessibility.

Also, don't use `<br>`s for line spacing; use margins. In this case it looks like your `<textarea>` could perform the spacing duties with top and bottom margins.